recipe/database.php

```diff
@@ -17,6 +17,26 @@ function () use ($databaseUtility) {
     }
 )->select('stage=staging');
 
+desc('Create the local database if it does not exists yet');
+task(
+    'sumo:db:create-local',
+    function () use ($databaseUtility) {
+        $localDatabaseUrl = parse_url(
+            Configuration::fromLocal()->get('DATABASE_URL')
+        );
+
+        runLocally(
+            sprintf(
+                'mysql %1$s -e "CREATE DATABASE IF NOT EXISTS %2$s; GRANT ALL PRIVILEGES ON %2$s.* TO %3$s@localhost IDENTIFIED BY \'%4$s\'"',
```

Comment:
Is die Grant nodig? 
En kan je niet `symfony console doctrine:database:create` aanroepen?

Reply:
GRANT is nog om de gebruiker toegang te geven, maar lokaal werken we altijd met root. Dus is op zich niet nodig.

Reply:
Opgekuist